submissions/sinchUK/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,204 @@
+                                    </div>
+                                    <div class="tooltip-item">
+                                        <a href="#">
+                                            <div class="tooltip-item-icon">
+                                                <img src="./icons/classroom.png" width="45" height="45" alt="Classroom"/>
+                                            </div>
+                                            <span>Classroom</span>
+                                        </a>
+                                    </div>
+                                    <div class="tooltip-item">
+                                        <a href="#">
+                                            <div class="tooltip-item-icon">
+                                                <img src="./icons/collection.png" width="45" height="45" alt="Collection"/>
+                                            </div>
+                                            <span>Collection</span>
+                                        </a>
+                                    </div>
+                                    <div class="tooltip-item">
+                                        <a href="#">
+                                            <div class="tooltip-item-icon">
+                                                <img src="./icons/mars.png" width="45" height="45" alt="Mars"/>
+                                            </div>
+                                            <span>Mars</span>
+                                        </a>
+                                    </div>
+                                    <div class="tooltip-item">
+                                        <a href="#">
+                                            <div class="tooltip-item-icon">
+                                                <img src="./icons/blogs.png" width="45" height="45" alt="Blogs"/>
+                                            </div>
+                                            <span>Blogs</span>
+                                        </a>
+                                    </div>
+                                    <div class="tooltip-item">
+                                        <a href="#">
+                                            <div class="tooltip-item-icon">
+                                                <img src="./icons/promo.png" width="45" height="45" alt="Promos"/>
+                                            </div>
+                                            <span>Promos</span>
+                                        </a>
+                                    </div>
+                                </diV>
+                            </div>
+                            <input class="tooltip-more__input  hidden" id="even-more-input" type="checkbox"/>
+                            <label class="tooltip-more-btn hidden more-btn" for="even-more-input">Even more from Kottans</label>
```

Comment:
Logically it's a link)

Reply:
Yes). 
![Screenshot 2022-10-20 135348](https://user-images.githubusercontent.com/17921586/196929717-768e9697-2375-4d44-9393-3ec9d928d570.png)

Reply:
Yes). 
![Screenshot 2022-10-20 141135](https://user-images.githubusercontent.com/17921586/196933366-70eead37-cddd-4e3b-8e7b-69624ae42a13.png)